doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/azure.md

```diff
@@ -0,0 +1,88 @@
+
+# Launching Ray Clusters on Azure
+
+This guide details the steps needed to start a Ray cluster on Azure.
+
+There are two ways to start an Azure Ray cluster.
+- Launch through Ray cluster launcher.
+- Deploy a cluster using Azure portal .
+
+```{note}
+Azure cluster configurations are community-maintained.
```

Comment:
```suggestion
Azure cluster configurations are community-maintained and are subject to change.
```

Personally, I don't really know what the implications of "community-maintained" are.

Reply:
```suggestion
The Azure integration is community-maintained. Please reach out to the integration maintainers on Github if
you run into any problems: gramhagen, eisber, ijrsvt.
```